examples/standalone/benchmarks/plotting.py

```diff
@@ -30,23 +30,34 @@
         key=lambda k: datetime.strptime(k["setup"]["timestamp"], "%d/%m/%Y %H:%M:%S")
     )
     for plottype in ["mainLoop", "initializationTotal"]:
-        keyval = (
-            ["main_loop"] if plottype == "mainLoop" else ["initialization", "total"]
+        data = (
+            ["mainloop", "DynCore", "Remapping", "TracerAdvection"]
+            if plottype == "mainLoop"
+            else ["initialization", "total"]
         )
         plt.figure()
         for backend in ["python/gtx86", "python/numpy", "fortran", "python/gtcuda"]:
             specific = [x for x in alldata if x["setup"]["version"] == backend]
             if specific:
-                for key in keyval:
+                for line in data:
                     plt.plot(
                         [
                             datetime.strptime(
                                 e["setup"]["timestamp"], "%d/%m/%Y %H:%M:%S"
                             )
                             for e in specific
                         ],
-                        [e["times"][key]["mean"] for e in specific],
-                        label=key + " " + backend,
+                        [
+                            e["times"][line]["mean"]
```

Comment:
What does e mean? Element?

Reply:
yes. I tend to use 1-letter abbreviations when doing list comprehensions for readabiliy

Reply:
To me this is really unclear and I would like to expand it to a word